zanata-war/src/main/java/org/zanata/exception/ProjectNotFoundException.java

```diff
@@ -0,0 +1,14 @@
+package org.zanata.exception;
```

Comment:
- [ ] If you haven't, you should set up your IDE to add header comments (LGPL, Red Hat 2016, etc) for new classes.  (Also VersionNotFoundException.java).